commit 3fb895ac88e99201573352b1abc18db4340ecede
Author: DJ Delorie <dj@redhat.com>
Date:   Sat Dec 21 23:12:41 2024 -0500

    assert: Use __writev in assert.c [BZ #32492]
    
    Replace writev with __writev in assert/assert.c.  This fixes [BZ #32492].
    
    Reviewed-by: H.J. Lu <hjl.tools@gmail.com>

diff --git a/assert/assert.c b/assert/assert.c
index 6002cc953cdb2d39..1e9683cf0707b0d3 100644
--- a/assert/assert.c
+++ b/assert/assert.c
@@ -115,7 +115,7 @@ __assert_fail_base (const char *fmt, const char *assertion, const char *file,
          this code path is taken.  */
       WS ("' failed\n");
 
-      (void) writev (STDERR_FILENO, v, i);
+      (void) __writev (STDERR_FILENO, v, i);
     }
 
   abort ();
